index.bs

```diff
@@ -325,8 +321,16 @@ interface PictureInPictureWindow {
 };
 </pre>
 
-The {{PictureInPictureWindow}} is a simple proxy to get information such as
-dimensions about the current Picture-in-Picture window.
+A {{PictureInPictureWindow}} instance represents a <a>Picture-in-Picture
+window</a>. It has a |state| that is either |opened| or |closed|.
```

Comment:
To elaborate on what's above, you could say that "when instantiated, an instance of PIPWindow has its state set to opened"

Reply:
Done.